submissions/evgeniy24/friends_app/app.js

```diff
@@ -0,0 +1,234 @@
+const REQUEST_URL  = 'https://randomuser.me/api/?results=20',
+    FRIENDS_LIST_WRAP = document.querySelector('.friendsList-wrap'),
+    MENU_WRAP = document.querySelector('.menu-wrap'),
+    SORTING_MENU = document.querySelector('.filter-sort-menu'),
+    SEARCH_FIELD = document.querySelector('#search'),
+    SORT_BY_NAME_AZ = document.querySelector('.nameSortAZ'),
+    SORT_BY_NAME_ZA = document.querySelector('.nameSortZA'),
+    SORT_BY_AGE_YO = document.querySelector('.ageSortYoungerFirst'),
+    SORT_BY_AGE_OY = document.querySelector('.ageSortOlderFirst'),
```

Comment:
Redundant because you can use `value` from checkboxes

Reply:
Done, refactored !